graphistry/embed_utils.py

```diff
@@ -0,0 +1,117 @@
+import torch
+import torch.nn as nn
+from torch.utils.data import Dataset, DataLoader
+
+class HeterographEmbedModuleMixin(nn.Module):
```

Comment:
not sure how we want to do class-level lazy imports, I think I had to do that elsewhere already for `nn`, but may have been sloppy just to push initial lazy loader through